Makefile

```diff
@@ -33,15 +33,17 @@ lint:
 	rm ${TEMP}/prometheus_alerts.yaml
 
 .PHONY: build
+.ONESHELL:
 build:
 	set -xe
 	helm repo add kedacore https://kedacore.github.io/charts
 	rm -rf ${TEMP}/astronomer || true
 	mkdir -p ${TEMP}
 	cp -R ../astronomer ${TEMP}/astronomer || cp -R ../project ${TEMP}/astronomer
-	# Install the external chart
-	helm dependency update ${TEMP}/astronomer/charts/postgresql
-	helm dependency update ${TEMP}/astronomer/charts/keda
+	# Install external charts
+	helm dep update ${TEMP}/astronomer/charts/postgresql
```

Comment:
Should we simplify this to `helm dep update`, or is there a reason that we're listing them one by one?

Reply:
i think it won't work.

Reply:
yeah, helm dep update at the top level isn't working. I can do a loop though.